README_fr.md

```diff
@@ -50,10 +42,8 @@ Jirafeau est un site web permettant d'héberger et de partager des fichiers de m
 
 ---
 
-Informations pour les développeurs
-----------------
+## Informations pour les développeurs
 
-**Seulement si vous voulez utiliser une branche de test pour le codage, au lieu de fusionner directement dans la banche principale.**
```

Comment:
As I understand, this statement in bold is for the app packager and it makes little sense to leave it. The other entries are empty, therefore not needed (Limitations, Configuration, Additional information). The Badge for Jessie seems to be outdated.

Reply:
This is for the contributors to the app, not for the packager.
Empty categories are still useful as it keep the template similar to all other apps. And so, if an info is added, it's easy to know where to put it.

Reply:
I was told in a previous pull request for another app to remove this as it was only for packagers...